sdk/nodejs/resource.ts

```diff
@@ -701,6 +701,14 @@ export interface ResourceOptions {
      * parents walking from the resource up to the stack.
      */
     transformations?: ResourceTransformation[];
+
+    /**
+     * Optional list of transforms to apply to this resource during construction. The
+     * transforms are applied in order, and are applied prior to transforms applied to
+     * parents walking from the resource up to the stack.
```

Comment:
Add a note that this option is experimental (like in `xRegisterStackTransform`)?